src/store/messagesStore.js

```diff
@@ -632,23 +672,47 @@ const actions = {
 			}
 			context.dispatch('processMessage', message)
 			if (!lastMessage || message.id > lastMessage.id) {
+				if (!message.systemMessage) {
+					countNewMessages++
+
+					// parse mentions data to update "conversation.unreadMention",
+					// if needed
+					if (!hasNewMention && hasMentionToSelf(context, message)) {
+						hasNewMention = true
+					}
+				}
 				lastMessage = message
 			}
+
+			// in case we encounter an already read message, reset the counter
+			// this is probably unlikely to happen unless one starts browsing from
+			// an earlier page and scrolls down
+			if (conversation.lastReadMessage.id === message.id) {
```

Comment:
conversation.lastReadMessage is an integer on the api?

Reply:
indeed, I thought it was `conversation.lastMessage` which is a full object.
fixing...